pkgs/servers/http/nginx/modules.nix

```diff
@@ -33,6 +33,29 @@ in
     inputs = [ pkgs.openssl ];
   };
 
+  auth-a2aclr = {
+    src = fetchFromGitLab {
+      name = "auth-a2aclr";
+      owner = "arpa2";
+      repo = "nginx-auth-a2aclr";
+      rev = "bbabf9480bb2b40ac581551883a18dfa6522dd63";
+      sha256 = "sha256-h2LgMhreCgod+H/bNQzY9BvqG9ezkwikwWB3T6gHH04=";
+    };
+    inputs = [
+      (pkgs.arpa2common.overrideAttrs
+        (old: rec {
+          version = "0.7.1";
```

Comment:
That version is two years old. Can't we use a newer one?

Reply:
The homepage https://gitlab.com/arpa2/nginx-auth-a2aclr says the requirements are:
> * source of liba2aclr 0.7.1 (ARPA2 Resource ACLs, part of libarpa2common)
> * source of nginx >=1.17

`arpa2common` has advanced quite a bit since, and I'm not sure if it's compatible anymore.

Reply:
Can you please check if newer versions still work?